app/main.js

```diff
@@ -915,6 +923,47 @@ function showNotification (text) {
   })
 }
 
+function runCommand (event, argv, workingDirectory) {
+  const args = argv.slice(app.isPackaged ? 1 : 2)
+  const cmd = new Command(args, app.getVersion())
+
+  // if this command is already executed by the second-instance, return early
+  if (!cmd.checkInMain()) return
+
+  switch (cmd.command) {
+    case 'reset':
+      resetBreaks()
+      break
+
+    case 'mini':
+      if (cmd.options.title) nextIdea = [cmd.options.title, '']
+      if (cmd.options.skip) skipToMicrobreak(false)
+      break
+
+    case 'long':
+      nextIdea = [cmd.options.title ? cmd.options.title : '', cmd.options.text ? cmd.options.text : '']
+      if (cmd.options.skip) skipToBreak(false)
+      break
+
+    case 'resume':
+      if (breakPlanner.isPaused) resumeBreaks(false)
+      break
+
+    case 'toggle':
+      if (breakPlanner.isPaused) resumeBreaks(false)
+      else pauseBreaks(1)
+      break
+
+    case 'pause':
+      var ms = cmd.durationToMs(settings)
+      pauseBreaks(ms)
+      break
+
+    default:
+      log.Error(`Unsupported command ${cmd.command}`)
```

Comment:
`Error` should be lower-case, I guess.

Reply:
https://github.com/papey/stretchly/commit/1e7b00595434ce69819118b9063385bdd1bea32a